src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/TranslatedDocument.java

```diff
@@ -10,6 +10,7 @@
 import uk.gov.hmcts.reform.civil.documentmanagement.model.DocumentType;
 
 import static uk.gov.hmcts.reform.civil.documentmanagement.model.DocumentType.DEFENCE_TRANSLATED_DOCUMENT;
+import static uk.gov.hmcts.reform.civil.documentmanagement.model.DocumentType.CLAIM_INTENTION_TRANSLATED_DOCUMENT;
```

Comment:
this should be CLAIMANT_INTENTION_TRANSLATED_DOCUMENT

Reply:
Fixed